woocommerce/wc-admin
# Extensibility: Add activity panel inbox tutorial

Adds a tutorial on how extension developers can add their own items to the activity panel inbox.

To test:

- Install docsify if you haven’t already:

	`npm i docsify-cli -g`

- If you needed to do that, you will want to bounce your terminal to get docsify on your path. Then, fire up a local server from the root of the wc-admin plugin.

	`docsify serve docs`

- And then point a browser at http://localhost:3000 to see the newly added Examples content.
